### ISSUE TITLE
Viewing Pods and Nodes has redundant content and possibly incorrect URL

### ISSUE THREAD

**Issue by fyu17** [kubernetes/website#41259]
The section "Show the app in the terminal" of the page [Viewing Pods and Nodes](https://kubernetes.io/docs/tutorials/kubernetes-basics/explore/explore-intro/) is a repetition of the section "View the app" of the previous page [Using kubectl to Create a Deployment](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/). In both cases, users are guided to start a proxy and do a curl of the same pod. 

The url of the section "Show the app in the terminal" seems to be broken. Despite the proxy running, the command `curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/proxy/` returns the following json:
```
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "error trying to reach service: dial tcp 10.244.0.4:80: connect: connection refused",
  "reason": "ServiceUnavailable",
  "code": 503
}
```
The url of the section in the previous page does not have the extra `proxy/` appended, and doing a curl properly returns the pod's config.

**Comment by devampatel03** [comment]
Hey @fyu17 ,  I would like to work  upon this. Can you please assign me the issue ?

**Comment by devampatel03** [comment]
Hey @fyu17 , I would like to work upon this. Can you please assign me the issue ?

**Comment by fyu17** [comment]
@devampatel03 I think there is already a fix for this #40590

**Comment by devampatel03** [comment]
Okay

**Comment by craigcomstock** [comment]
I do think that the issue I submitted is a duplicate of this one. It would seem the problem is that the `/proxy/` ending URL is trying to access the pod at port 80 instead of 8080 where the javascript server is listening.

**Comment by craigcomstock** [comment]
Here's my attempt at a fix. I noticed in https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster-services/#manually-constructing-apiserver-proxy-urls that the port could be specified so I tried that and things "work".

Still, though, later on at https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/ it seems that things aren't "quite right" but maybe that's another issue.
